src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/ClusterAwareConnectionProxy.java

```diff
@@ -104,10 +106,9 @@
   /** The logger we're going to use. */
   protected transient Log log = NULL_LOGGER;
 
-  protected static final int DEFAULT_SOCKET_TIMEOUT_MS = 10000;
-  protected static final int DEFAULT_CONNECT_TIMEOUT_MS = 30000;
   protected static final int NO_CONNECTION_INDEX = -1;
-  protected static final int WRITER_CONNECTION_INDEX = 0; // writer host is always stored at index 0
+  protected static final int WRITER_CONNECTION_INDEX = 0;
+  // writer host is always stored at index 0
```

Comment:
This comment should on previous line, or before it. Otherwise it's not clear what it corresponds to.